src/olympia/migrations/1084-remove-discovery-modules-table.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE IF EXISTS `discovery_modules`;
```

Comment:
need a new line at the end of the file (that's the red icons shown on github at the end of the line)

Reply:
Thank you @eviljeff . I fixed it again.